dolphinscheduler-master/src/main/java/org/apache/dolphinscheduler/server/master/engine/executor/plugin/subworkflow/SubWorkflowLogicTask.java

```diff
@@ -222,6 +234,11 @@ private SubWorkflowLogicTaskRuntimeContext triggerNewSubWorkflow() {
         final ICommandParam commandParam =
                 JSONUtils.parseObject(workflowInstance.getCommandParam(), ICommandParam.class);
 
+        final List<Property> paramList = mergeParams(asList(
+                commandParam.getCommandParams(),
+                new ArrayList<>(deserializeVarPool(workflowInstance.getVarPool())),
+                new ArrayList<>(deserializeVarPool(workflowInstance.getGlobalParams()))));
```

Comment:
The varpool's priority should be highest.
```suggestion
                new ArrayList<>(deserializeVarPool(workflowInstance.getGlobalParams()))),
                commandParam.getCommandParams(),
                new ArrayList<>(deserializeVarPool(workflowInstance.getVarPool())));
```

Reply:
Force pushed again because I committed a compile-time error. The varpool should have the highest priority now when merging parameters.